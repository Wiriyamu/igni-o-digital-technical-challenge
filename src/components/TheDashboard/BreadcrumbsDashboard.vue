<template>
  <section class="breadcrumbs">
    <div class="breadcrumbs__left">
      <p class="breadcrumbs__results">Resultados</p>
      <img src="@/assets/images/divider.svg" alt="Divisor" />
      <img
        class="breadcrumbs__icons-left"
        src="@/assets/images/home.svg"
        alt="Home"
      />
      <img
        class="breadcrumbs__icons-left"
        src="@/assets/images/chevrons-right.svg"
        alt="Flechas"
      />
      <p class="breadcrumbs__course">Cursos</p>
    </div>

    <div class="breadcrumbs__right">
      <img
        class="breadcrumbs__settings"
        src="@/assets/images/settings.svg"
        alt="Configurações"
      />
    </div>
  </section>
</template>

<script>
export default {
  name: 'BreadcrumbsDashboard'
}
</script>

<style lang="scss" scoped>
@import '@/assets/sass/abstracts/_variables.scss';

.breadcrumbs {
  display: flex;
  justify-content: space-between;
  margin: 2rem 0;

  @media screen and (max-width: 800px) {
    flex-wrap: wrap;
  }

  &__left {
    display: flex;
    align-items: center;

    @media screen and (max-width: 800px) {
      flex-wrap: wrap;
    }
  }

  &__results {
    margin-right: 3rem;
    font-size: 2.4rem;
  }

  &__icons-left,
  &__course {
    margin-left: 1.5rem;
  }

  &__right {
    display: flex;
    justify-content: center;
    align-items: center;
    width: 4rem;
    height: 4rem;
    border-radius: 4rem;
    cursor: pointer;
    background: #ec008c;
    background: -webkit-linear-gradient(to right, #fc6767, #ec008c);
    background: linear-gradient(to right, #fc6767, #ec008c);
  }

  &__settings {
    width: 1.8rem;
  }
}
</style>
