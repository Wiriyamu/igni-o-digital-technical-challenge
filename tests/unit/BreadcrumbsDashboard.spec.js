import BreadcrumbsDashboard from '@/components/TheDashboard/BreadcrumbsDashboard'
import { mount } from '@vue/test-utils'

describe('BreadcrumbsDashboard.vue', () => {
  it('Snapshot component', () => {
    const wrapper = mount(BreadcrumbsDashboard)

    expect(wrapper.html()).toMatchSnapshot()
  })
})
