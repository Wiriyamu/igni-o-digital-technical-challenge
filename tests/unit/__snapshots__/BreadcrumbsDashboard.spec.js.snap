// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BreadcrumbsDashboard.vue Snapshot component 1`] = `
<section class="breadcrumbs">
  <div class="breadcrumbs__left">
    <p class="breadcrumbs__results">Resultados</p><img src="" alt="Divisor"><img class="breadcrumbs__icons-left" src="" alt="Home"><img class="breadcrumbs__icons-left" src="" alt="Flechas">
    <p class="breadcrumbs__course">Cursos</p>
  </div>
  <div class="breadcrumbs__right"><img class="breadcrumbs__settings" src="" alt="Configurações"></div>
</section>
`;
